src/Binkoeff.java

```diff
@@ -0,0 +1,39 @@
+
+public class Binkoeff {

```

Comment:
Sieht lustig aus, gäbe in der Semesterarbeit aber richtig Abzüge für kryptische und unnötige Abkürzung des Klassennamen... 😜